internal/connect/command_offset_status_describe.go

```diff
@@ -0,0 +1,160 @@
+	Message         string `human:"Message,omitempty"`
+	AppliedAt       string `human:"Applied At,omitempty"`
+	PreviousOffsets string `human:"Previous Offsets,omitempty"`
+}
+
+func (c *offsetCommand) newStatusDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Describe connector offset update or delete status.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.statusDescribe,
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Describe the status of the latest offset update/delete operation for a connector in the current or specified Kafka cluster context.",
+				Code: "confluent connect offset status describe lcc-123456",
+			},
+			examples.Example{
+				Code: "confluent connect offset status describe lcc-123456 --cluster lkc-123456",
+			},
+		),
+	}
+
+	pcmd.AddClusterFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *offsetCommand) statusDescribe(cmd *cobra.Command, args []string) error {
+	kafkaCluster, err := kafka.GetClusterForCommand(c.V2Client, c.Context)
+	if err != nil {
+		return err
+	}
+
+	environmentId, err := c.Context.EnvironmentId()
+	if err != nil {
+		return err
+	}
+
+	connector, err := c.V2Client.GetConnectorExpansionById(args[0], environmentId, kafkaCluster.ID)
+	if err != nil {
+		return err
+	}
+
+	connectorName := connector.Info.GetName()
+
+	offsetStatus, err := c.V2Client.AlterConnectorOffsetsRequestStatus(connectorName, environmentId, kafkaCluster.ID)
+	if err != nil {
+		return err
+	}
+
+	if output.GetFormat(cmd) == output.Human {
+		return printHumanDescribeOffsetStatus(cmd, offsetStatus, args[0])
+	}
+
+	return printSerializedDescribeOffsetStatus(cmd, offsetStatus, args[0])
+}
+
+func printHumanDescribeOffsetStatus(cmd *cobra.Command, offsetStatus connectv1.ConnectV1AlterOffsetStatus, id string) error {
+	var appliedAt string
+	if offsetStatus.AppliedAt.IsSet() {
+		appliedAt = offsetStatus.AppliedAt.Get().String()
+	}
+
+	var phase string
+	var message string
+	_, isStatusSet := offsetStatus.GetStatusOk()
+	if isStatusSet {
+		phase = offsetStatus.GetStatus().Phase
+		if messagePtr := offsetStatus.GetStatus().Message; messagePtr != nil {
+			message = *messagePtr
+		}
+	}
+
+	var prevOffsets []map[string]any
+	var offsetStr string
+	if offsetStatus.HasPreviousOffsets() {
+		prevOffsets = *offsetStatus.PreviousOffsets
+		offSetBytes, err := json.Marshal(prevOffsets)
+		if err != nil {
+			return err
+		}
+		offsetStr = string(pretty.Pretty(offSetBytes))
+	}
+
+	table := output.NewTable(cmd)
+	table.Add(&humanOffsetStatusDescribeOut{
+		Id:              id,
+		RequestType:     string(offsetStatus.Request.Type),
+		RequestedAt:     offsetStatus.Request.GetRequestedAt().String(),
+		Phase:           phase,
+		Message:         message,
+		AppliedAt:       appliedAt,
+		PreviousOffsets: offsetStr,
```

Comment:
For PATCH type we should also show the what the requested update in offsets was.

Reply:
Done